keyrings/alt/file_base.py

```diff
@@ -10,6 +10,7 @@
 from keyring.backend import KeyringBackend
 from keyring.util import platform_, properties
 from keyring.util.escape import escape as escape_for_ini
+from keyring.py27compat import string_types
```

Comment:
Probably better to use `six.string_types`, since that's available.

Reply:
Done.